__tests__/intergration.tests.js

```diff
@@ -56,6 +56,40 @@ describe("non existent endpoint", () => {
   });
 });
 
+describe("/api/articles", () => {
+  test("GET:200 returns status 200", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+
+  test("GET:200 returns an arrayof all articles without body property", () => {
+    return request(app)
+      .get("/api/articles")
+
+      .then((response) => {
+        response.body.articles.rows.forEach((article) => {
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.body).toBe("undefined");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.article_img_url).toBe("string");
+        });
```

Comment:
We will also need a comment count to be included. This can be done using a join in your SQL :)

Reply:
Sorry, my bad. I just didn't see it there.